routes/userSchema.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const Boom = require('boom');
+const UserController =require('../controller/user');
+const Validator = require('../schema/userSchema');
+exports.register = function (server, options, next) {
+    // instantiate controller
+    const userController = new UserController(server.db);
+
+    server.bind(userController);
+
+    // GET USER API
+    server.route({
```

Comment:
you call the controller from userSchema, do you think is that correct?

Reply:
I just changed it